lib/implementations/batch/batch.js

```diff
@@ -11,7 +11,7 @@ module.exports = class Batch extends Builder.Flow {
     if (new.target === Batch) {
       throw new Error('Cannot construct Batch instances directly');
     }
-    this._storeOptions = _.pick(options, ["number", "groupBy", "attributes", "timeout", "type", "path"]);
+    this._storeOptions = _.pick(options, ["number", "groupBy", "attributes", "timeout", "type", "path", "timestamp"]);
```

Comment:
passed time stamp parameter should be treated similar to group by or attribute parameters